index.html

```diff
@@ -10,20 +10,33 @@
 <body>
     <header class="header">
              <a href="#">Gleen</a>
-          
 
-        <nav>
-            <img src="./Images/Union.png" alt="hamburger">
-        
-            <ul class="desktop-links">
-                <li><a href="#Portfolio">Portfolio</a></li>
-                <li><a href="#About">About</a></li>
-                <li><a href="#Contact">Contact</a></li>
-            </ul>
-        </nav>
-    </header>
-
-    <section class="headline">
+             <div class="menu">
+            
+                <div id="myNav" class="overlay">
+              
+                      <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+              
+                  <div class="overlay-content">
+                      <a href="#Portfolio">Portfolio</a>
+                      <a href="#About">About</a>
+                      <a href="#Contact">Contact</a>
+                  </div>
```

Comment:
[OPTIONAL]

- [ ] I would recommend you apply the same function that you used on the **times icon** to close the mobile menu on each of the menus(`onclick="closeNav()`).  Relying on HTML attributes might not be so effective 😄 .